### PR TITLE
Simple Date Formatting for Draft/Session List Items

### PR DESCRIPTION
**Pull request by AbhinavThukral97**
This PR provides a simple function to convert the date formatted as 'DD/MM/YYYY' to one these relevant strings: 'Today', 'x days ago', 'x months ago' or 'x years ago'.
Note: 'x minutes/x seconds' cannot be calculated yet as the only input received is the formatted date of creation.

### CODE REVIEW

File: frog/imports/client/GraphEditor/Graph.js [hunk 1]
```diff
@@ -22,8 +22,8 @@ const scrollMouse = e => {
 };
 
 const mousemove = e => {
-  // We do -100 here because there is 100px above the graph editor
-  store.ui.socialMove(e.clientX, e.clientY - 100);
+  // Subtracting Graph Editor's position from the top of the screen
```
Comment (houshuang) [inline frog/imports/client/GraphEditor/Graph.js]: This should not be part of this PR.

**Review by houshuang — CHANGES_REQUESTED** [review]

**Comment by houshuang** [comment]
@AbhinavThukral97 	Note that currently, the parsing of dates is done in UserDashboard/data-utils/helpers.js:

```
export const parseDate = (date): Date => {
  return `${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`;
};
```

this function takes a Date object that also has time. I think it would make sense to move your logic here, and also include minutes ago?

File: frog/imports/client/GraphEditor/Graph.js (hunk 1 above)
Comment (AbhinavThukral97) [reply frog/imports/client/GraphEditor/Graph.js]: Ah yeah I branched this by mistake. I'll fix this.

**Comment by AbhinavThukral97** [comment]
I see, I'll check this out.
